Form1.cs

```diff
@@ -53,5 +53,22 @@ private void textBox1_TextChanged(object sender, EventArgs e)
         {
 
         }
+        //  int Input, Expenses, Goal, Months;
+           // String goalName = nameGoal.Text;
+           // Input = int.Parse(InputBox.Text);
+           // Expenses = int.Parse(ExpensesBox.Text);
+          //  Goal = int.Parse(GoalBox.Text);
+           // Months = Goal/(Input - Expenses);
+           // MessageBox.Show(goalName + " remaining time is " + Months.ToString() + " months");
     }
+    
+            private void button2_Click(object sender, EventArgs e)
+        {
+          int Input, Expenses, Goal, Months;
```

Comment:
It is better to name local variables in lowercase (as you named goalName)

Reply:
+